app/js/containers/SignatureContainer.jsx

```diff
@@ -0,0 +1,35 @@
+import React, {Component, Fragment} from 'react';
+import {connect} from 'react-redux';
+import IncludeSignatureForm from '../components/IncludeSignatureForm';
+import PropTypes from 'prop-types';
+import signature from '../features/signature';
+
+class SignatureContainer extends Component {
+  includeSignature = (signature) => {
+    this.props.includeSignature(signature);
+  };
+
+  render() {
+    return <Fragment>
+      <IncludeSignatureForm onSubmit={this.includeSignature} result={this.props.receipt} error={this.props.error} />
```

Comment:
no need to rename receipt in result,
A result is very generic while we all know what is a receipt

Especially that it can be mixed with arbitration result.